app/services/appCivistService.js

```diff
@@ -46,6 +46,12 @@ appCivistApp.factory('Assemblies', function ($resource, localStorageService, $in
     assembly: function (assemblyId) {
       return $resource(getServerBaseUrl(localStorageService) + '/assembly/:aid', { aid: assemblyId }, { 'update': { method: 'PUT' } });
     },
+    uploadMembers: function (assemblyId, send, payload) {
+      return $resource(getServerBaseUrl(localStorageService) + '/assembly/:aid/member?send_invitations=:send', {
```

Comment:
@gvescu this has to be `POST /api/assembly/{aid}/campaign/{cid}/group/{gid}/member`